README.md

```diff
@@ -1,8 +1,10 @@
 # VueTorrent WebUi - Docker mod for qBittorrent
 
-This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start.
+This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start. 
```

Comment:
please change installed to installed/updated as you confirmed it also updates

Reply:
If changing to installed/updated; should I remove the second sentence or change it in any way?

Reply:
I would so it follows the standard the rest use.